### PR TITLE
add process_id to handshake, refactor status enum into bools named content_analysis_request and verdict_success.

### PR DESCRIPTION
**Pull request by eric-waldman**

### CODE REVIEW

**Comment by eric-waldman** [comment]
Made an update replacing the optional `status` Enum with an optional boolean named `success`. It's a bit cleaner this way.

File: proto/content_analysis/sdk/analysis.proto [hunk 1]
```diff
@@ -8,24 +8,24 @@ option optimize_for = LITE_RUNTIME;
...
-  }
-  optional Status status = 1; 
+  // Process ID of the browser initiating the session.
+  optional int32 process_id = 1;
+  // Indicates whether a content analysis request will be sent in this session,
+  // following the handshake. If set to True, then the browser is expecting to
+  // receive a verdict response from the agent in this session, following
+  // the request. If set to False, the browser is performing a startup check or
+  // an intermittent heartbeat, and no response is expected from the agent.
+  optional bool has_content_analysis_request = 2;
```
Comment (nancy-lan-xiao) [inline proto/content_analysis/sdk/analysis.proto:20]: Consider avoiding using has_ as prefix, see https://yaqs.corp.google.com/eng/q/1274890329476038656#a2 for explanation.

Comment (eric-waldman) [reply proto/content_analysis/sdk/analysis.proto:20]: nice! thanks for the tip.

File: proto/content_analysis/sdk/analysis.proto [hunk 2]
```diff
@@ -8,24 +8,24 @@ option optimize_for = LITE_RUNTIME;
...
-  }
-  optional Status status = 1; 
+  // Process ID of the browser initiating the session.
+  optional int32 process_id = 1;
+  // Indicates whether a content analysis request will be sent in this session,
+  // following the handshake. If set to True, then the browser is expecting to
+  // receive a verdict response from the agent in this session, following
+  // the request. If set to False, the browser is performing a startup check or
+  // an intermittent heartbeat, and no response is expected from the agent.
+  optional bool content_analysis_request = 2;
```
Comment (nancy-lan-xiao) [inline proto/content_analysis/sdk/analysis.proto:20]: Nit: you might want to add an adjective or a past participles to the end for clarification. See https://g3doc.corp.google.com/engage/g3doc/standards/protos.md?cl=head#field_names

e.g. content_analysis_request ->  content_analysis_request_sent

File: proto/content_analysis/sdk/analysis.proto [hunk 3]
```diff
@@ -8,24 +8,24 @@ option optimize_for = LITE_RUNTIME;
...
+// from the agent.
 message Acknowledgement {
-  enum Status {
-    FAILURE = 0;
-    SUCCESS = 1;
-  }
-  optional Status status = 1;
+  // Indicates whether a verdict was successfully acted on by the browser.
+  // This should be logged by agents, especially if it's False.
+  optional bool verdict_success = 1;
```
Comment (nancy-lan-xiao) [inline proto/content_analysis/sdk/analysis.proto:28]: Same here, verdict_success -> verdict_succeeded  or  verdict_acted_on

**Review by nancy-lan-xiao — CHANGES_REQUESTED** [review]

File: proto/content_analysis/sdk/analysis.proto (hunk 2 above)
Comment (eric-waldman) [reply proto/content_analysis/sdk/analysis.proto:20]: Thanks for the link to the proto naming convenctions, havent' nseen them before!

Changed it to content_analysis_requested.

File: proto/content_analysis/sdk/analysis.proto (hunk 3 above)
Comment (eric-waldman) [reply proto/content_analysis/sdk/analysis.proto:28]: Changed to verdict_received.

**Review by nancy-lan-xiao — APPROVED** [review]

**Pull request merged**